### DISCUSSION TITLE
Stop loss Question

### DISCUSSION THREAD

**Discussion by traderblakeq** [kernc/backtesting.py#944] [General]
Hi,

![image](https://user-images.githubusercontent.com/36839989/227519357-8d59b725-8fe5-4cf5-bf68-3a461ffc6b0f.png)

-  I go short on price = 100  & set SL = 150
-  Price goes against me, and the recorded loss in PnL result will be 200.

My question is:

Is there a way to override SL, So I can ensure that the backtest result does record SL=150, and not the next candle Close value (200) in PnL?.

**Comment by kernc** [comment]
If stop-loss is not applied in the same candle the trade was opened in, this seems like a bug. I think there might be something about it on the issue tracker already ...
